src/basho_bench_stats_writer_csv.erl

```diff
@@ -204,7 +204,8 @@ stringify_stats(_RunStatsType=csv, RunMetrics) ->
             [StrVal|Acc]
         end, [], UniqueHeaders),
 
-        [io_lib:format("~s\n", [string:join([atom_to_list(Op)|CsvLine],", ")])|Lines]
+        [io_lib:format("~s\n", 
+            [string:join( [atom_to_list(Op) | CsvLine], ", ")]) | Lines]
```

Comment:
Or a combination where you have `OpLine = io_lib:format("~s\n", [string:join( [atom_to_list(Op) | CsvLine], ", ")]),` and then `[OpLine | Lines]`.